dateparser/data/date_translation_data/nb.py

```diff
@@ -93,11 +93,13 @@
     ],
     "week": [
         "uke",
-        "u"
+        "u",
+        "uker"
```

Comment:
:lipstick: I know this is not currently the case, but it may be a good idea to take this opportunity to include a comma in the last item as well, and maybe even sort the entries alphabetically. Same for the rest of the changes.

```suggestion
        "uker",
```